src/app/features/profile/profile-list/profile-list.component.html

```diff
@@ -0,0 +1,13 @@
+<mat-list *ngFor="let user of users">
+
+    <mat-list-item>
+        <div class="text" (click)="toProfileDetail(user)">
+            <img [src]="user.picture"/>
+            {{ user.firstName }} {{ user.lastName }}
+        </div>
+    </mat-list-item>
+    <div *ngIf="user.displayed">
+        <app-profile-detail [user]="user" ></app-profile-detail>
```

Comment:
the requirements were "sending the user to a user details page with that user data"
also it is not very common to use another page container as a child component for the other page
for example must have practice is to use ChangeDetection.OnPush
but this strategy is often skipped for the page container components while you need it here